docs/sidebars.js

```diff
@@ -242,6 +242,13 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "FAQs",
+      items: [
+        "articles/testkube-licensing-FAQ",
```

Comment:
what title will this entry have in the menu? the first headline from the doc above?

Reply:
You can find it at the Netlify preview, I think it is the title of the page:

<img width="333" alt="image" src="https://github.com/kubeshop/testkube/assets/142315352/e1efa1b9-15ba-4303-b2ab-bd51936d00b2">